core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/ConfigResource.java

```diff
@@ -111,12 +111,16 @@ public ConfigUpdateRequest updateSubjectLevelConfig(
   public Config getSubjectLevelConfig(@PathParam("subject") String subject) {
     Config config = null;
     try {
+      if(!schemaRegistry.hasSubjectConfig(subject)) {
```

Comment:
Could just check
`if(compatibilityLevel == null)` (after calling getCompatibilityLevel of course)

If checking for null is distasteful, note that having null check here is consistent with the other Rest api methods, and I think consistency counts.